ActiveDataProvider.php

```diff
@@ -32,6 +32,17 @@ class ActiveDataProvider extends \yii\data\ActiveDataProvider
      */
     private $_queryResults;
 
+    /**
+     * @inheritdoc
+     */
+    public function __construct(array $config = [])
+    {
+        /** `totalCount` is calculated after the request */
+        if (!empty($config['pagination'])) {
+            $config['pagination']['totalCount'] = false;
```

Comment:
Should be mentioned that obtaining count is a cheap operation in ElasticSearch so it makes sense to always calculate it.